util.go

```diff
@@ -40,6 +40,8 @@ func (v VersionFlag) BeforeApply(app *Kong, vars Vars) error {
 // early in the parsing process, changing how other flags resolve relative paths.
 //
 // Use this flag to provide a "git -C" like functionality.
+//
+// It is not compatible with custom named decoders, e.g., existingdir.
```

Comment:
Not sure where my comment here went.

But this is because it seems custom named decoders have precedence over `Decode` function of a custom type.